infogami/infobase/__init__.py

```diff
@@ -1,10 +1,14 @@
 """
 Infobase.
 """
+from __future__ import print_function
+
+import logging
```

Comment:
Appears to be unused

Reply:
[flake8](http://flake8.pycqa.org) testing of https://github.com/internetarchive/infogami on Python 3.7.1

$ __flake8 . --select=F401 --statistics__
```
./test/bug_239238.py:11:1: F401 'infogami.infobase.infobase' imported but unused
./infogami/config.py:5:1: F401 'web' imported but unused
./infogami/plugins/links/code.py:9:1: F401 'web' imported but unused
./infogami/plugins/links/code.py:15:1: F401 'view' imported but unused
./infogami/plugins/i18n/code.py:8:1: F401 'infogami.config' imported but unused
./infogami/plugins/i18n/code.py:10:1: F401 'infogami.utils.context.context' imported but unused
./infogami/plugins/i18n/code.py:11:1: F401 'infogami.utils.view.public' imported but unused
./infogami/plugins/i18n/code.py:12:1: F401 'infogami.utils.template.render' imported but unused
./infogami/plugins/i18n/db.py:2:1: F401 'infogami.infobase.client' imported but unused
./infogami/infobase/logger.py:13:1: F401 'time' imported but unused
./infogami/infobase/logger.py:79:9: F401 'web' imported but unused
./infogami/infobase/dbstore.py:8:1: F401 'collections.defaultdict' imported but unused
./infogami/infobase/dbstore.py:12:1: F401 '_dbstore.schema.Schema' imported but unused
./infogami/infobase/dbstore.py:12:1: F401 '_dbstore.schema.INDEXED_DATATYPES' imported but unused
./infogami/infobase/common.py:1:1: F401 'config' imported but unused
./infogami/infobase/account.py:7:1: F401 'simplejson' imported but unused
./infogami/infobase/_dbstore/read.py:3:1: F401 'collections.defaultdict' imported but unused
./infogami/infobase/tests/test_store.py:6:1: F401 'simplejson' imported but unused
./infogami/infobase/tests/conftest.py:2:1: F401 'pytest_wildcard.*' imported but unused
./infogami/infobase/tests/test_seq.py:4:1: F401 'unittest' imported but unused
./infogami/infobase/tests/test_seq.py:5:1: F401 'simplejson' imported but unused
./infogami/infobase/tests/test_client.py:3:1: F401 'infogami.infobase.server' imported but unused
./infogami/utils/types.py:13:5: F401 'web' imported but unused
./infogami/utils/app.py:5:1: F401 're' imported but unused
./tests/test_doctests.py:2:1: F401 'doctest' imported but unused
25    F401 'web' imported but unused
```